SETUP.md

```diff
@@ -23,17 +23,16 @@ This document describes how to setup all the dependencies to run the notebooks i
 
 Depending on the type of recommender system and the notebook that needs to be run, there are different computational requirements. Currently, this repository supports the following environments:
 
-* Python CPU
-* Python GPU
-* PySpark
+* Python CPU / GPU
+* PySpark CPU / GPU
 
 ## Setup guide for Local or DSVM
 
 ### Requirements
 
 * Machine running Linux, Windows Subsystem for Linux ([WSL](https://docs.microsoft.com/en-us/windows/wsl/about)) or macOS
 * Anaconda with Python version >= 3.6.
-  * This is pre-installed on Azure DSVM, for local setup [Miniconda](https://docs.conda.io/en/latest/miniconda.html) is a quick way to get started.
+  * This is pre-installed on Azure DSVM. For local setup [Miniconda](https://docs.conda.io/en/latest/miniconda.html) is a quick way to get started.
```

Comment:
Perhaps change "This is pre-installed on Azure DSVM" to "This is pre-installed on Azure DSVM such that one can run the following steps directly". The reasons is previously someone tries the repo after installing miniconda manually and that leads to some path issue.

Reply:
make sense